include/bluetooth/mesh/gen_plvl_srv.h

```diff
@@ -168,10 +171,17 @@ struct bt_mesh_plvl_srv {
 	struct bt_mesh_plvl_range range;
 	/** Current Default Power. */
 	uint16_t default_power;
-	/** The last known Power Level. */
-	uint16_t last;
-	/** Whether the Power is on. */
-	bool is_on;
+	struct {
+		/** The last known Power Level. */
+		uint16_t last;
+		/** Whether the Power is on. */
+		bool is_on;
```

Comment:
All these parameters are just a judgment. But it follows last state and I based it a little bit on light where it is going on off. It will change often if someone is switching something on and off. And it is symmetric with light_lightness.